app/views/hyrax/homepage/_home_content.html.erb

```diff
@@ -1,5 +1,5 @@
-<div class="col-xs-12 col-sm-6">
-  <ul id="homeTabs" class="nav nav-pills">
+<div class="col-sm-6">
```

Comment:
Was the change to not make it full width on mobile intentional?

Reply:
I think when using any `col-sm-*`, full-width is implied for mobile sizing.

Reply:
👌